hack/verify-vulnerabilities.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
another option would be to rename the `verify-container-images.sh` to maybe `verify-vulnerabilities.sh` and integrate `make govulncheck` into that.